app/routes/faq.tsx

```diff
@@ -15,11 +15,34 @@ export default function FAQ() {
 		<Layout back title="FAQ">
 			<Heading level={2}>Frequently Asked Questions</Heading>
 
-			<Heading level={3}>What is the parking situation at your events?</Heading>
+			<Heading level={3}>Can I attend if I don't know TypeScript?</Heading>
 			<BodyText>
-				When meeting at the Microsoft NERD Center, you can park along the water
-				in front of the entrance, or in pay-to-park nearby. There is also a paid
-				parking garage in the building.
+				Absolutely! We prep our speakers to know not all attendees will know it.
+				We'd love to have you join our community regardless of experience.
+			</BodyText>
+
+			<Heading level={3}>
+				How do I get to your events by public transit?
+			</Heading>
+			<BodyText>
+				We recommend taking the Red Line to the Kendall/MIT Station, which is a
+				five minute walk away from the{" "}
+				<a href="https://maps.app.goo.gl/7TMY5xxytpEHh6CX7">
+					Microsoft NERD center
+				</a>
+				.
+				<br />
+				There are also several e-bike stations around the building.
+			</BodyText>
+
+			<Heading level={3}>How do I park at your events?</Heading>
+			<BodyText>
+				When meeting at the{" "}
+				<a href="https://maps.app.goo.gl/7TMY5xxytpEHh6CX7">
+					Microsoft NERD center
+				</a>
```

Comment:
Thoughts about:

```suggestion
				<AnchorWithArrow href="https://maps.app.goo.gl/7TMY5xxytpEHh6CX7">
					Microsoft NERD center
				</AnchorWithArrow>
```

Missing at the top:

```ts
import { AnchorWithArrow } from "~/components/AnchorWithArrow";
```

(probably good to add it to the "parkacarincambridge" link as well)